app/scripts/store/dimFarming.directive.js

```diff
@@ -16,9 +16,9 @@
             <img class="engram" ng-class="{ active: (vm.service.movingItems || vm.service.makingRoom) }" src="/images/engram.svg" height="60" width="60"/>
           </span>
           <span>
-            <p>DIM is moving Engram and Glimmer items from {{vm.service.store.name}} to the vault and leaving one space open per item type to prevent anything from going to the Postmaster.</p> 
+            <p>DIM is moving Engram and Glimmer items from {{vm.service.store.name}} to the vault and leaving one space open per item type to prevent anything from going to the Postmaster.</p>
             <div class="item-details"><span>
-              <p><span translate="configuration"></span></p>
```

Comment:
Changing it to `translate="::configuration"` would have also worked. I believe it is technically better to use the directives before using the brackets notation. 

Brackets are dirty checked and refreshed in every $digest even if it is not necessary.

Not trying to call you out: this is an issue throughout ALL of DIM, and is just a coding style that we've adopted.

Not asking you to change it, but going forward I wonder if we should consider this.

All that being said, I don't think that angular will pick up things in the curly brackets if you did bind-once (so maybe it would be fine this way anyway.)

@bhollis @SunburnedGoose thoughts? Am I mistaken?